tests/test_configurations.py

```diff
@@ -0,0 +1,31 @@
+def test_configurations():
+    import os
+    from yucca.training.configuration.configure_callbacks import get_callback_config
+    from yucca.training.configuration.configure_paths_and_plans import get_path_and_plan_config
+    from yucca.training.configuration.input_dimensions import get_input_dims
+    from yucca.training.configuration.split_data import get_split_config
+    from yucca.paths import yucca_preprocessed_data
+
+    path_and_plan_config = get_path_and_plan_config(task="Task000_Test", model_dimensions="2D", model_name="TinyUNet")
+    assert path_and_plan_config is not None
+
+    input_dims_config = get_input_dims(
+        plan=path_and_plan_config.plans,
+        model_dimensions=path_and_plan_config.model_dimensions,
+        num_classes=path_and_plan_config.num_classes,
+        model_name=path_and_plan_config.model_name,
+        batch_size="tiny",
+        patch_size="tiny",
+    )
+    assert input_dims_config is not None
+
+    split_config = get_split_config(train_data_dir=path_and_plan_config.train_data_dir, task=path_and_plan_config.task)
+    assert split_config is not None
+
+    callback_config = get_callback_config(
+        task=path_and_plan_config.task,
+        save_dir=path_and_plan_config.save_dir,
+        version_dir=path_and_plan_config.version_dir,
+        version=path_and_plan_config.version,
+    )
+    assert callback_config is not None
```

Comment:
We should be able to make some more asserts here on the contents of the configs. There are many use cases and ideally each should have its own test case. At a minimum I would suggest:

1. Split the test case up into one for each of path+plan, input dims, split, callbacks. 
2. Test the content of the config (eg. does the path_and_plan_config contain the right save_dir?)

Reply:
good call